src/dotnet-roslyn-tools/Commands/DartTestCommand.cs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the License.txt file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.RoslynTools.Utilities;
+using Microsoft.RoslynTools.VS;
+
+namespace Microsoft.RoslynTools.Commands;
+
+internal static class DartTestCommand
+{
+    private static readonly string[] s_allProductNames = VSBranchInfo.AllProducts.Select(p => p.Name.ToLower()).Concat(new[] { "all" }).ToArray();
```

Comment:
No need to include `all` since this can only run against a single product at a time.